sklearn/model_selection/tests/test_validation.py

```diff
@@ -1715,6 +1719,8 @@ def test_score():
 
     def two_params_scorer(estimator, X_test):
         return None
-    fit_and_score_args = [None, None, None, two_params_scorer]
+
+    scorer = _MultimetricScorer(score=two_params_scorer)
+    fit_and_score_args = [None, None, None, scorer]
```

Comment:
Shouldn't the original list `[None, None, None, two_params_scorer]` still be tested?

Reply:
This is testing a private method `_score`. On master, a multimetric scoring was represented with a dictionary, which `_score` used to call the scorers independently. This PR moves this responsibility from `_score` to `_MultimetricScorer`. Now `_score` only needs to call `_MultimetricScorer.__call__`.